### DISCUSSION TITLE
Access to LPUART1 (pins 24/25)

### DISCUSSION THREAD

**Discussion by giacomocariello** [mciantyre/teensy4-rs#151] [Q&A] [answered]
I'm trying to use LPUART1 (pins 24/25). It's not explicitly mapped in teensy4-bsp, however I noticed that the HAL/RAL parts seem available in imxrt-rs. I tried to create the LPUART1 manually, but its interrupt never gets triggered. Any clue on how to proceed? Should I open an issue to request a new feature?

Thanks in advance for any help, it's really appreciated.

**Comment by mciantyre** [comment, marked as answer]
#152 shows the changes that I think we need to use LPUART1 with pins 24 and 25. I don't have the hardware to test those changes, but I included two examples that use this LPUART configuration. Could you let me know if that works for you?

**Reply by giacomocariello** [reply]
I confirm that it works fine. As a side note, I'd be glad to sponsor hardware needs for the teensy4-bsp project. If you don't mind, I'd get in touch through Matrix Chat for the details.

**Reply by mciantyre** [reply]
LPUART1 is available in teensy4-bsp 0.4.6.

Thanks for the offer. I'm not taking Teensy 4-specific hardware donations right now. However, I'd consider taking inventory of i.MX RT EVKs, or hardware that supports the larger imxrt-rs organization. My commits include an email address.
